helpers/bajasHelper/bajasHelper.helper.go

```diff
@@ -366,3 +370,66 @@ func TraerDetalle(id int) (Solicitud map[string]interface{}, outputError map[str
 		return nil, outputError
 	}
 }
+
+func GetDetalleElemento(id int) (Elemento *models.DetalleElementoBaja, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{"funcion": "/GetDetalleElemento", "err": err, "status": "502"}
```

Comment:
```suggestion
			outputError = map[string]interface{}{"funcion": "/GetDetalleElemento - Unhandled Error!", "err": err, "status": "500"}
```
Se recomienda combinar esto con el uso de utils_oas/errorctrl para no repetir tanto código